VDF.GUI/Utils/PickerDialogUtils.cs

```diff
@@ -19,21 +19,46 @@
 namespace VDF.GUI.Utils {
 	internal static class PickerDialogUtils {
 
+		private static string GetLocalPath(Uri uriPath) {
+			string localPath = uriPath.LocalPath;
+			if (uriPath.IsFile) {
+				// The `LocalPath` member of the `Uri` object we got
+				// from `TryGetUri()` might be wrong if the original
+				// path contains the '#' character, or if escape
+				// sequences (e.g. '%61') occur in the path.
+				//
+				// Thus, it's better to use the `OriginalString` member
+				// instead, which contains the correct data on both
+				// Linux and Windows.  However, the `file://` scheme
+				// might be prefixed (e.g. on Linux), so it needs to be
+				// removed if present.
+				// See https://github.com/0x90d/videoduplicatefinder/pull/400
+				string scheme = uriPath.Scheme + Uri.SchemeDelimiter;
```

Comment:
Can it be made const (if not static readonly) and moved out of method?

Reply:
Since the scheme is determined from the `uriPath` (this way we don't have to do assumption about it), I think we can't ?

Reply:
You're right sorry, I read it as `Uri.Scheme` for some reason